kindlyops/mappamundi
# Closes HAV-62

Signed-off-by: Kevin N. Murphy <knmurphy@users.noreply.github.com>
# Description

This pulls in the JavaScript libraries we were using in page but loading from third-party CDNs. Now they are included in this repo and loaded from local filesystem. This allows for less third-party dependancies to run Haven. 

## Discussion list or pending tasks


## How to test

Run haven, check out various pages and inspect to see if any JS libraries are 404
